stabilizer/src/main/java/com/hazelcast/stabilizer/agent/workerjvm/WorkerJvmLauncher.java

```diff
@@ -124,10 +129,27 @@ private WorkerJvm startWorkerJvm(String mode) throws IOException {
         new WorkerJvmProcessOutputGobbler(process.getInputStream(), new FileOutputStream(logFile)).start();
         workerJvm.process = process;
         workerJvm.mode = WorkerJvm.Mode.valueOf(mode.toUpperCase());
+        uploadResourcesToWorker(workerId);
         workerJVMs.put(workerId, workerJvm);
         return workerJvm;
     }
 
+    private void uploadResourcesToWorker(String workerId) throws IOException {
+        final String testSuiteId = agent.getTestSuite().id;
+        if(!new File(workersPath + "/" + testSuiteId + "/resources/").exists()){
+            log.info("Resources files are not in the agent");
```

Comment:
I would log this under debug since a user perhaps has not specified the upload directory. Also I would improve the message because it is very confusing.